aws-ec2-manager-blueprint-inputs.yaml

```diff
@@ -1,23 +1,40 @@
-aws_access_key_id: ''
-aws_secret_access_key: ''
-image_id: ''
-instance_type: ''
-manager_keypair_name: ''
-agent_keypair_name: ''
-
-use_existing_manager_group: false
-use_existing_agent_group: false
-use_existing_manager_keypair: false
-use_existing_agent_keypair: false
-ssh_key_filename: ~/.ssh/cloudify-manager-kp.pem
-agent_private_key_path: ~/.ssh/cloudify-agent-kp.pem
-manager_security_group_name: cloudify-manager-security-group
-agent_security_group_name: cloudify-agent-security-group
-manager_server_name: cloudify-manager-server
-ssh_user: centos
-agents_user: centos
+# Credentials and identification in order to connect to ec2
+#aws_access_key_id: ''
+#aws_secret_access_key: ''
+#image_id: ''
+#instance_type: ''
+
+# These are the key names in ec2 that will be created at bootstrap.
+# If existing key pairs are used (see above), bootstrap will use the provided names, and will not create new ones.
```

Comment:
maybe "see below"? the manager and agent keypair names are below, if that's what you referred to